modules/core/src/main/java/org/apache/ignite/internal/processors/security/GridSecurityProcessor.java

```diff
@@ -77,7 +89,7 @@
     public SecuritySubject authenticatedSubject(UUID subjId) throws IgniteCheckedException;
 
     /**
-     * Gets security context.
+     * Gets security context for authenticated nodes and thin clients.
```

Comment:
Let's at least extend javadoc of SecurityContext in order to explain its area of responsibility.
It deserved something more than this:
```
/**
 * Security context.
 */
public interface SecurityContext {
```

Reply:
ok